package.json

```diff
@@ -100,7 +101,10 @@
     "@changesets/cli": "^2.13.1",
     "@commitlint/cli": "16.2.1",
     "@commitlint/config-conventional": "16.2.1",
-    "@emotion/babel-preset-css-prop": "^10.0.14",
+    "@emotion/babel-preset-css-prop": "^11.2.0",
+    "@emotion/css": "^11.9.0",
+    "@emotion/eslint-plugin": "^11.7.0",
```

Comment:
`@emotion/eslint-plugin` is the new ESLint plug in for emotion v11. This package replaces `eslint-plugin-emotion`.